.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,69 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+
+---
+
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+<!--- Thank you for keeping this note for the community --->
+
+### Affected Module Name(s):
+
+<!--- Please list the affected resources or module names. --->
+
+* aci_XXXXX
+
+### APIC version and APIC Platform
+
+* V x.x.x and on-prem/cloud-aws/cloud-azure/all.
+
+### Affected Resource(s)
```

Comment:
Why do we have this twice as we already have the affected modules above. Remove this section.